test with issues

resolve:  should throw named error: last testcase.
